### PR TITLE
EOS-25493: Added cron and gatekeepers email

### PR DESCRIPTION
**Pull request by venkuppu-chn**
Signed-off-by: Venkatesh K <venkatesh.k@seagate.com>

# Problem Statement
Cron and gatekeepers list is missing in the alex workflow

# Design
Added the cron and gatekeepers list

# Coding
   Checklist for Author
-  [x] Coding conventions are followed and code is consistent

# Testing 
  Checklist for Author

- [x] Unit and System Tests are added
http://eos-jenkins.colo.seagate.com/job/Release_Engineering/job/re-workspace/job/Alex-Scan/52/
- [ ] Test Cases cover Happy Path, Non-Happy Path and Scalability
- [ ] Testing was performed with RPM

# Impact Analysis
  Checklist for Author/Reviewer/GateKeeper
- [ ] Interface change (if any) are documented
- [ ] Side effects on other features (deployment/upgrade)
- [ ] Dependencies on other component(s)

# Review Checklist 
  Checklist for Author
- [x] JIRA number/GitHub Issue added to PR
- [x] PR is self reviewed
- [x] Jira and state/status is updated and JIRA is updated with PR link
- [x] Check if the description is clear and explained

# Documentation
  Checklist for Author
- [ ] Changes done to WIKI / Confluence page / Quick Start Guide

### CODE REVIEW

**Review by shailesh-vaidya — DISMISSED** [review]

File: jenkins/automation/alex.groovy [hunk 1]
```diff
@@ -143,7 +153,7 @@ pipeline {
               script {
                   def useEmailList = ''
                   if ( params.environment == 'prod') {
-                    useEmailList = 'shailesh.vaidya@seagate.com, priyank.p.dalal@seagate.com, john.bent@seagate.com, venkatesh.k@seagate.com'
+                    useEmailList = 'nilesh.govande@seagate.com, basavaraj.kirunge@seagate.com, ivan.tishchenko@seagate.com, john.bent@seagate.com, venkatesh.k@seagate.com'
```
Comment (gauravchaudhari02) [inline jenkins/automation/alex.groovy:156]: You can remove nilesh govande as he is not in cortx

Comment (venkuppu-chn) [reply jenkins/automation/alex.groovy]: Thanks Gaurav. I have removed him from the list

**Review by gauravchaudhari02 — APPROVED** [review]

**Comment by venkuppu-chn** [comment]
Jenkins Job:
http://eos-jenkins.colo.seagate.com/job/Release_Engineering/job/re-workspace/job/Alex-Scan/52/

File: jenkins/automation/alex.groovy [hunk 2]
```diff
@@ -127,7 +137,7 @@ pipeline {
               script {
                   def useEmailList = ''
                   if ( params.environment == 'prod') {
-                    useEmailList = 'shailesh.vaidya@seagate.com, priyank.p.dalal@seagate.com, john.bent@seagate.com, venkatesh.k@seagate.com'
+                    useEmailList = 'shailesh.vaidya@seagate.com, mukul.malhotra@seagate.com, john.bent@seagate.com, venkatesh.k@seagate.com'
```
Comment (mukul-seagate11) [inline jenkins/automation/alex.groovy:140]: can all the email addressed be removed as its internal & update/store in parameter

**Review by mukul-seagate11 — CHANGES_REQUESTED** [review]

**Comment by shailesh-vaidya** [comment]
@venkuppu-chn Could you please address review comments for this PR?

**Comment by shailesh-vaidya** [comment]
@venkuppu-chn Please revolve conflicts and address review comments.

**Comment by shailesh-vaidya** [comment]
@venkuppu-chn Is this PR still valid? If yes could you please resolve conflicts and address comments.

**Pull request closed without merging**